templates/patterns/list_item/pattern-list-item--variant-navigation.html.twig

```diff
@@ -2,12 +2,30 @@
 /**
  * @file
  * Default implementation for the navigation list item component.
+ *
+ * @see @ecl/navigation-lists
  */
 #}
-{% include '@ecl/ec-component-list-item' with {
-  'variant': 'navigation',
-  'title': title,
-  'href': url,
-  'detail': detail,
-  'screen_reader': 'List item'|t
+
+{% set _infos = {} %}
```

Comment:
The infos are not used in the navigation variant.

Reply:
good catch.